velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -236,6 +257,10 @@ VeloxToSubstraitExprConvertor::toSubstraitNotNullLiteral(
       literalExpr->set_boolean(variantValue.value<TypeKind::BOOLEAN>());
       break;
     }
+    case velox::TypeKind::SMALLINT: {
```

Comment:
Let's group all integer cases together, i.e. TINYINT, SMALLINT, INTEGER, BIGINT.

Reply:
I fixed this.